app/routers/world_clock.py

```diff
@@ -0,0 +1,744 @@
+    'Europe/Saratov': {'country': 'Russia', 'subcountry': 'Saratov'},
+    'Europe/Simferopol': {'country': 'Ukraine',
+                          'subcountry': 'Crimea'},
+    'Europe/Sofia': {'country': 'Bulgaria',
+                     'subcountry': 'Sofia-Capital'},
+    'Europe/Stockholm': {'country': 'Sweden',
+                         'subcountry': 'Stockholm'},
+    'Europe/Tallinn': {'country': 'Estonia', 'subcountry': 'Harjumaa'},
+    'Europe/Ulyanovsk': {'country': 'Russia', 'subcountry': 'Ulyanovsk'},
+    'Europe/Vienna': {'country': 'Austria', 'subcountry': 'Vienna'},
+    'Europe/Vilnius': {'country': 'Lithuania',
+                       'subcountry': 'Vilnius County'},
+    'Europe/Volgograd': {'country': 'Russia', 'subcountry': 'Volgograd'},
+    'Europe/Warsaw': {'country': 'Poland',
+                      'subcountry': 'Masovian Voivodeship'},
+    'Europe/Zurich': {'country': 'United States',
+                      'subcountry': 'Illinois'},
+    'Indian/Mahe': {'country': 'India',
+                    'subcountry': 'Madhya Pradesh'},
+    'Pacific/Apia': {'country': 'Brazil', 'subcountry': 'Sao Paulo'},
+    'Pacific/Auckland': {'country': 'United Kingdom',
+                         'subcountry': 'England'},
+    'Pacific/Chatham': {'country': 'United Kingdom',
+                        'subcountry': 'England'},
+    'Pacific/Funafuti': {'country': 'Tuvalu', 'subcountry': 'Funafuti'},
+    'Pacific/Gambier': {'country': 'Australia',
+                        'subcountry': 'South Australia'},
+    'Pacific/Guam': {'country': 'Brazil', 'subcountry': 'Pará'},
+    'Pacific/Honolulu': {'country': 'United States',
+                         'subcountry': 'Hawaii'},
+    'Pacific/Majuro': {'country': 'Marshall Islands',
+                       'subcountry': 'Majuro Atoll'},
+    'Pacific/Norfolk': {'country': 'Canada', 'subcountry': 'Ontario'},
+    'Pacific/Pago_Pago': {'country': 'American Samoa',
+                          'subcountry': 'Eastern District'},
+    'Pacific/Palau': {'country': 'Mexico', 'subcountry': 'Coahuila'},
+    'Pacific/Port_Moresby': {'country': 'Papua New Guinea',
+                             'subcountry': 'National Capital'},
+    'Pacific/Tarawa': {'country': 'Kiribati',
+                       'subcountry': 'Gilbert Islands'},
+    'Pacific/Wake': {'country': 'United Kingdom', 'subcountry': 'England'},
+}
+CONTINENTS = ['Africa', 'America', 'Antarctica', 'Asia',
+              'Australia', 'Europe', 'Indian', 'Pacific']
+CITY_NAME_KEY = "name"
+COUNTRY_NAME_KEY_IN_CONTITNENTS = "Country_Name"
+COUNTRY_NAME_KEY_IN_CITIES = "country"
+SUBCOUNTRY_NAME_KEY_IN_CITIES = "subcountry"
+CONTINENT_NAME_KEY = "Continent_Name"
+CONTINENT = 'continent'
+COUNTRY = 'country'
+PLACE = 'place'
+CURRENT_TIME_KEY = 'datetime'
+TIMEZONE_PARTS_MAP = {CONTINENT: 0, PLACE: -1}
+TIME_SCOPE_INDEX = 0
+FEEDBACK_INDEX = 1
+ARB_COUNTRY_TIMEZONE = -1
+PATH_SEPARETOR = '/'
+PARTS_OF_THE_DAY_FEEDBACK = {'Early morning':
```

Comment:
It would probably be better to use NamedTuple here

Reply:
Can you explain why It would be better? I find the dict more convenient and elegant here... :)